docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -1066,31 +1381,73 @@ and checks to see if the result is greater than 1.
+    in performance is negligible for small data sets but more pronounced when working with large numbers
+    of nodes.
 
+*Filter the current working set of network flows (inet:flow nodes) to only include flows where the total number of bytes transferred in the flow between the source (inet:flow:src:txbytes) and destination (inet:flow:dst:txbytes) is greater than 100MB (~100,000,000 bytes):*
+
+.. storm-pre:: [ ( inet:flow=* :src:txbytes=60000000 :dst:txbytes=60000000 ) ( inet:flow=* :src:txbytes=1024 :dst:txbytes=4096 ) ] +$( :src:txbytes + :dst:txbytes >=100000000 )
+::
+  
+  <inet:flow> +$( :src:txbytes + :dst:txbytes >=100000000 )
 
-- From the set of nodes associated with any threat group or threat cluster (e.g., tagged
-  ``#cno.threat.<threat_name>``), filter results to those nodes that are attributed to more than one
-  threat (e.g., that have more than one ``#cno.threat.<threat_name>`` tag. This may identify nodes that
-  are incorrectly attributed to more than one group; or instances where two threat clusters overlap,
-  which may indicate that the clusters actually represent a single set of activity):
+
+*Filter the current set of nodes associated with any threat group or threat cluster (e.g., tagged ``#cno.threat.<threat_name>``), to include only those nodes that are attributed to more than one threat (e.g., that have more than one #cno.threat.<threat_name> tag):*
 
 .. storm-pre:: [ ( inet:ipv4=4.4.4.4 +#cno.threat.foo ) ( inet:ipv4=7.7.7.7 +#cno.threat.hurr +#cno.threat.derp ) ] +$( $node.globtags(cno.threat.*).size() > 1 )
 ::
   
   #cno.threat +$( $node.globtags(cno.threat.*).size() > 1 )
 
-This example uses the :ref:`meth-node-globtags` method to select the set of tags on each node that
-match the specified expression (``cno.threat.*``) and :ref:`stormprims-list-size` to count the number
-of matches.
+This query may identify nodes that are incorrectly attributed to more than one group, or instances where
+two or more threat clusters overlap (which may indicate that the clusters actually represent a single set
+of activity).
+
+This example uses the :ref:`meth-node-globtags` method to select the set of tags on each node that match
+the specified expression (``cno.threat.*``) and :ref:`stormprims-list-size` to count the number of matches.
+
+*Filter the current working set of DNS A records (inet:dns:a nodes) to only include those whose .seen interval falls WITHIN the past 30 day window (e.g., where the <min> value of the .seen interval is greater than or equal to the date 30 days in the past:*
+
+.. storm-pre:: [ ( inet:dns:a = ( woot.com, 4.4.4.4 ) .seen = now ) ] $now = $lib.time.now() $30d = 2592000000 $start = $( $now - $30d ) inet:dns:a.seen @= ( $now, -30 days ) ( $min, $max ) = .seen +$( $min >= $start )
+::
+  
+  $now = $lib.time.now()
+  $30d = 2592000000
+  $start = $( $now - $30d )
+  inet:dns:a.seen @= ( $now, -30 days )
```

Comment:
Not sure if this messes up your example, but you could do:
```
$start = $lib.cast(time, -30days)
inet:dns:a.seen @= ( $start,  now )
```

or maybe
```
$ival = $lib.cast(ival, (now, -30days))
($start, $stop) =  $ival
inet:dns:a.seen @= $ival
```
To avoid two separate "-30days" time calculations (which while extremely unlikely could potentially not match up)

Reply:
Nice! Updated.